code/datums/progressbar.dm

```diff
@@ -63,7 +63,7 @@
 	spawn(5)
 		if(client)
 			client.images -= bar
-		qdel(bar)
+		//qdel(bar) //ChompEDIT - try not qdelling progressbars.
```

Comment:
Not setting it t null?
Or is this already fine? :v

Reply:
SHIT you right, one sec